docs/configuration.md

```diff
@@ -284,6 +286,33 @@ Then issue `:CocConfig` and add the following to your Coc config file.
 }
 ```
 
+##### Example with Settings
+
+```json
+{
+  "languageserver": {
+    "haskell": {
+      "command": "haskell-language-server-wrapper",
+      "args": ["--lsp"],
+      "rootPatterns": [ "*.cabal", "stack.yaml", "cabal.project", "package.yaml", "hie.yaml" ],
+      "filetypes": ["haskell", "lhaskell"],
+      "disabledFeatures": ["codeLens"],
```

Comment:
Maybe we don't need to include this setting since it's quite "generic" as far as CoC is concerned? It's really the haskell-specific settings that need a HLS-specific example, I guess?

Reply:
Good point of view. `disableFeatures` is a setting provided by CoC and like others should be documented well and looked up in CoC's docs. Focussing on HLS here makes sense. Dropping the line.